src/Value/Size.php

```diff
@@ -186,7 +186,7 @@ public function isColorComponent()
     /**
      * Returns whether the number stored in this Size really represents a size (as in a length of something on screen).
      *
-     * @return false if the unit an angle, a duration, a frequency or the number is a component in a Color object.
```

Comment:
I think just change this comment to not involve a type annotation - i.e. "Returns false if the unit [is]..." (and add the missing "is") - so the meaning is retained.